com.github.muriloventuroso.givemelyrics.json

```diff
@@ -0,0 +1,39 @@
+{
+    "app-id": "com.github.muriloventuroso.givemelyrics",
+    "runtime": "org.gnome.Platform",
+    "runtime-version" : "3.30",
+    "base" : "io.elementary.BaseApp",
+    "base-version" : "juno",
+    "sdk": "org.gnome.Sdk",
+    "command": "com.github.muriloventuroso.givemelyrics",
+    "finish-args": [
+        "--share=ipc",
+        "--share=network",
+        "--socket=x11",
+        "--device=dri",
+        "--socket=wayland",
+        "--socket=system-bus",
```

Comment:
This permission is way too broad you need to run your app with --log-system-bus or --log-session-bus as arguments and then whitelist the specific calls you need.